src/extension.ts

```diff
@@ -97,14 +97,8 @@ export function activate(context: vscode.ExtensionContext) {
                 vscode.commands.executeCommand('appService.Refresh', node.getParentNode());
             } catch (err) {
                 if (!(err instanceof UserCancelledError)) {
-                    try {
-                        // Azure REST error messages come as a JSON string with more details
-                        outputChannel.appendLine(JSON.parse(err.message).Message);
```

Comment:
Can you add this logic into `initAsyncCommand`? We still prefer to display `JSON.parse(err.message).Message`

Reply:
I believe the logic is in the util.errToString function to parse messages correctly.